js/HintView.js

```diff
@@ -0,0 +1,57 @@
+import Adapt from 'core/js/adapt';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { templates } from 'core/js/reactHelpers';
+import notify from 'core/js/notify';
+
+export default class HintView extends Backbone.View {
+
+    className() {
+        return 'hint';
+    }
+
+    initialize() {
+        this.onHintPopupClicked = this.onHintPopupClicked.bind(this);
+
```

Comment:
```suggestion
       this.$parentEl = $(`[data-adapt-id=${this.model.get('_id')]`)
```

Then reuse `this.$parentEl` throughout. The jQuery selector function `$()` has to search through the dom every time you lookup a selector to find all of the matching elements.

Please use [`data-adapt-id`](https://github.com/adaptlearning/adapt-contrib-core/blob/de6fcd9c58ad7ed0d793920b048fcb213bdcc980/js/views/adaptView.js#L15) instead of the class.